Sources/ComposableArchitecture/ViewStore.swift

```diff
@@ -436,6 +436,7 @@ private struct HashableWrapper<Value>: Hashable {
     ///   - action: An action.
     ///   - predicate: A predicate on `State` that determines for how long this method should
     ///     suspend.
+    @MainActor
```

Comment:
Swift 5.7 no longer guarantees that this method runs on the main actor, but this is required by SwiftUI.